ghdocs/TUTORIAL.md

```diff
@@ -268,49 +290,211 @@ Go ahead and open up `dist/components/List/List.html` from your extracted Fabric
 </div>
 ```
 
-Now, let's remove the background colors we used to test our two container `<divs> (.TodoList, .TodoBar)`. Their styles should now look like the following:
+Now, let's remove the background colors ( `background-color: #000000;`, `background-color: #cdcdcd;`) we used to test `<div class="TodoBar"></div>` and `<div class="TodoList></div>`. **_Your app.css file should look like the following:
```

Comment:
I think the last sentence is supposed to look a bit different than  "**_Your app.css file should..."